### ISSUE TITLE
Music not playing 

### ISSUE THREAD

**Issue by Standlord** [AsoDesu/IslandUtils#18]
The music is not playing. I hear nothing. Only at the start the "Overtime" (10 seconds) is all I can hear. Please help me to fix it

**Comment by Standlord** [comment]
i closed it accidentaly

**Comment by LukynkaCZE** [comment]
Do you have music enabled in settings and music turned on in music settings?

**Comment by Standlord** [comment]
I have turned on everything, here is a screenshot, only the pre-game "Overtime" jingle is everything I hear
![2022-09-14_17 24 43](https://user-images.githubusercontent.com/113526900/190196909-591b3eb2-1505-4cd1-b554-92e2b21e7aa5.png)

**Comment by Standlord** [comment]
Still no music for me after the Hotterfix 1.1.2 :(

**Comment by FayneAldan** [comment]
> I have turned on everything, here is a screenshot, only the pre-game "Overtime" jingle is everything I hear ![2022-09-14_17 24 43](https://user-images.githubusercontent.com/113526900/190196909-591b3eb2-1505-4cd1-b554-92e2b21e7aa5.png)

Normally the "Jukebox/Note Blocks" option would be replaced with "MCC Music":

![](https://media.essential.gg/11be7890-85bb-4076-bfd8-047326b60c00)

Are you using Fabric and have Fabric API installed? Can you send your Minecraft log?

**Comment by Standlord** [comment]
Yes, I'm using Fabric and have the API installed as well. Here's my latest minecraft log if you're looking for this
[latest.log](https://github.com/AsoDesu/IslandUtils/files/9572893/latest.log)

**Comment by FayneAldan** [comment]
> [19:09:27] [Render thread/INFO]: Connecting to play.MCCISLAND.net, 25565

`play.MCCISLAND.net`
https://github.com/AsoDesu/IslandUtils/blob/main/src/main/java/net/asodev/islandutils/state/MccIslandState.java#L29

Island Utils assumes that the IP address is all lowercase, like it should be. ;)

**Comment by Standlord** [comment]
Thank you for helping me it's perfect now :))
